### PR TITLE
Content Update including 2024 releases and react-google-maps

### PR DESCRIPTION
**Pull request by chrisgervang**
- add redirects for vis.gl projects w/o domains (a request from the Google Maps team)
  - i.e. from the URL https://vis.gl/react-google-maps/ to the URL https://visgl.github.io/react-google-maps/
- add react-google-maps and recent blog posts
- 9.0 release added to homepage
- footer and homepage mobile styles

### CODE REVIEW

File: content/frameworks.yaml [hunk 1]
```diff
@@ -75,12 +87,6 @@ frameworkGroups:
         description: >
           A high-performance feature editing framework for deck.gl.
 
-      - name: math.gl
```
Comment (ibgreen) [inline content/frameworks.yaml:78]: We should probably 
- replace nebula.gl with deck.gl-community
- add probe.gl?

**Review by ibgreen — APPROVED** [review]
Good improvements, thanks!

**Comment by ibgreen** [comment]
One thought is: how hard would it be to fold this content into openvisualization.org and link vis.gl to that site? Also, does openvisualization.org support markdown?

**Review by Pessimistress — APPROVED** [review]

**Comment by chrisgervang** [comment]
> One thought is: how hard would it be to fold this content into openvisualization.org and link vis.gl to that site? Also, does openvisualization.org support markdown?

I think most of the effort go into designing the new pages to look great.

From a technical side, these sites are built similarly. Openvis doesn't support markdown, instead its hooked into a no-code CRM called prismic.io to update content (its pretty nice).

The code's here: https://github.com/openjs-foundation/open-visualization

If we could get a design, I'd be down to implement it. We could set `vis.gl` to redirect to something like `openvisualization.org/vis.gl`

**Pull request merged**